templates/errors/403.html

```diff
@@ -1,8 +1,11 @@
 {% extends 'layouts/_base.html' %}
 
-{% set page_title = _("Authorisation refused") + " - " + _('Office for National Statistics') %}
+{% set page_title = _("Sorry, there is a problem") ~ " - " ~ _('Census 2021') %}
 
 {% block main %}
 <h1>{{ _("Sorry, there is a problem") }}</h1>
-<p>{{ _("To access this page you need to <a href='#'>enter your unique access code.</a>") }}</p>
+<p>{{ _("You may need to update your browser to a newer version.") }}</p>
+<p>{{ _("If the problem still occurs, try using a different browser or device.") }}</p>
+<p>{{ _("You may need to update your browser to a newer version.") }}</p>
```

Comment:
Is this supposed to be twice in this block?

Reply:
Nope removed, good spot.